src/views/CronTabDetails.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+
+import Loading from '@crontab-utils/Loading';
+import {
+  HorizontalNav,
+  K8sResourceCommon,
+  useK8sWatchResource,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { Bullseye } from '@patternfly/react-core';
+
+import CronTabPageTitle from './CronTabDetails/CronTabPageTitle';
+import CronTabDetailsPage from './CronTabDetailsPage';
+import CronTabYAMLPage from './CronTabYAMLPage';
+import { useCronTabTranslation } from '@crontab-utils/hooks/useCronTabTranslation';
+
+type CronTabPageProps = {
+  name: string;
+  namespace: string;
+  kind: string;
+};
+
+const CronTabNavPage: React.FC<CronTabPageProps> = ({ name, namespace, kind }) => {
+  const { t } = useCronTabTranslation();
+  const [cronTab, loaded] = useK8sWatchResource<K8sResourceCommon>({
+    groupVersionKind: {
+      group: 'stable.example.com',
+      kind: 'CronTab',
+      version: 'v1',
+    },
+    kind,
+    name,
+    namespace,
+  });
+
+  const pages = React.useMemo(
+    () => [
+      {
+        href: '',
+        name: t('Details'),
+        component: CronTabDetailsPage,
+      },
+      {
+        href: 'yaml',
+        name: t('YAML'),
+        component: CronTabYAMLPage,
+      },
+    ],
+    [],
```

Comment:
I think we need `t` as a dependency here to force re-render when the language changes?

Since others are using this as a model, we should check what's right here.